feature/announcement/src/main/java/io/github/droidkaigi/confsched2022/feature/announcement/AnnouncementsViewModel.kt

```diff
@@ -32,4 +33,10 @@ class AnnouncementsViewModel @Inject constructor(
             AnnouncementsUiModel(data)
         }
     }
+
+    fun onRetryButtonClick() {
+        viewModelScope.launch {
+            // TODO: Re-acquire data.
```

Comment:
We were unable to implement the process of reacquisition.🥺
I need help with this part.🙇‍♂️